bodo/tests/test_df_lib/test_end_to_end.py

```diff
@@ -2392,3 +2392,195 @@ def test_fallback_wrapper_deep_fallback():
     assert "TypeError triggering deeper fallback" in warning_msg, (
         f"Unexpected warning message: {warning_msg}"
     )
+
+
+import benchmarks.tpch.dataframe_lib as tpch
+from benchmarks.tpch.dataframe_lib import _query_to_datasets, collect_datasets
+
+datapath = "bodo/tests/data/tpch-test_data/parquet"
+
+
+def load_tpch_data(name, datapath):
+    if name == "lineitem":
+        return tpch.load_lineitem(datapath, pd=pd)
```

Comment:
Can replace the whole function with:
` return getattr(tpch, f"load_{name}")(datapath, pd=pd)`

Reply:
I wouldn't worry about the raise on an invalid tpch source name.